portal/config/eproms/Organization.json

```
@@ -1824,6 +1824,34 @@
       },
       "resourceType": "Organization"
     },
+    {
+      "extension": [
+        {
+          "timezone": "America/Denver",
```

I think this [timezone](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) is technically more accurate, but makes no real difference
```suggestion
          "timezone": "America/Edmonton",
```